### PR TITLE
Added Forgot password Page

### PR DESCRIPTION
**Pull request by Ankur9669**

### CODE REVIEW

File: common.css [hunk 1]
```diff
@@ -19,14 +20,17 @@ ul{
...
 footer{
     display: flex;
     flex-direction: column;
     justify-content: center;
     align-items: center;
     background-color: var(--primary-color);
     width: 100%;
-    min-height: 200px;
+    min-height: 12.5rem;
     color: white;
```
Comment (zoef-yt) [inline common.css]: Use var colour

**Review by zoef-yt — APPROVED** [review]
Good to merge with changes.

**Review by sri-rishi — APPROVED** [review]
except suggestion given above, Code and UI looks fine to merge

File: common.css [hunk 2]
```diff
@@ -4,12 +4,13 @@ body{
     background-color: var(--background-color);
     font-family: poppins;
     overflow-x: hidden;
-    max-width: 2000px;
+    max-width: 125rem;
     margin: auto;
 }
 :root{
     --light-blue: #f2fbff;
     --color-black: black;
```
Comment (parvez-rashid) [inline common.css]: Better to use hex here as well for consistency.

**Review by parvez-rashid — APPROVED** [review]
I've added a comment, otherwise looks great.

File: common.css (hunk 1 above)
Comment (Ankur9669) [reply common.css]: Done

File: common.css (hunk 2 above)
Comment (Ankur9669) [reply common.css]: Done

**Pull request merged**